test/foundry/unit/StakingRewardsV2/StakingV2RewardCalculations.t.sol

```diff
@@ -112,7 +112,7 @@ contract StakingV2RewardCalculationTests is DefaultStakingV2Setup {
         rewards = rewardEscrowV2.escrowedBalanceOf(user1);
         rewardsUsdc = usdc.balanceOf(user1);
         assertEq(rewards, expectedRewards);
-        assertEq(rewardsUsdc, expectedRewards);
+        assertGe(rewardsUsdc, expectedRewards);
```

Comment:
add upper bound as well if `assertEq` cannot work

Reply:
Improved this